GitX.xcodeproj/project.pbxproj

```diff
@@ -1746,7 +1746,7 @@
 				INFOPLIST_OTHER_PREPROCESSOR_FLAGS = "-traditional";
 				INSTALL_PATH = /Applications;
 				MACOSX_DEPLOYMENT_TARGET = 10.11;
-				PRODUCT_BUNDLE_IDENTIFIER = net.phere.GitX;
+				PRODUCT_BUNDLE_IDENTIFIER = io.github.GitX;
```

Comment:
How important is this? It means that settings from older versions will be lost, right? (There aren't that many settings, so that's probably not a big deal.) Would this affect things like "Open recent..." or anything else?

Reply:
I've no opinion, I just rebased it and I can't test this artifact. My son has my old x86 mac.
Maybe you can test your doubts ?
Should we merge or revert this line ?

Reply:
🤦 Of course; these automatic builds make this easy!

I've confirmed that this loses all saved preferences including the list of recent repos. While the prefs aren't too much, there are enough of them to be inconvenient: custom `git`, whitespace diff prefers, avatar, etc.

There was some previous discussion of this way back in #18. The summary of that was changing the ID would mostly benefit in by preventing another fork from reawakening and interfering w/ this one ... but that seems increasingly less likely now 5 years later.

> Should we merge or revert this line ?

I would say revert.

Reply:
Done, I reverted this